rasa/core/actions/action.py

```diff
@@ -283,6 +288,51 @@ def __str__(self) -> Text:
         return "ActionUtterTemplate('{}')".format(self.name())
 
 
+class EndToEndAction(Action):
```

Comment:
this is the essential part 3 of the change

Reply:
shall we name it in the spirit of other actions? Smth like `ActionEndToEndText`? or smth similar